.gitignore

```diff
@@ -9,3 +9,4 @@ cypress-coverage
 cypress/screenshots
 cypress/videos
 cypress/fixtures/example.json
+.vscode
```

Comment:
Why was this added?

Reply:
That's just for me and my `.vscode` extensions. I can remove it if you like. I thought I had pushed that up to the other one but, I hadn't should definitely do it to all of them or none of them. I'm just trying to avoid accidently committing a change there